code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -530,6 +530,7 @@
 		if(sheets.len > 0)
 			var/obj/item/stack/sheet/S = pick(sheets)
 			S.amount++
+			S.update_custom_materials(2000, 1)
```

Comment:
```suggestion
			S.update_custom_materials()
```
This doesn't actually use any arguments. It just recalculates the amount of materials from scratch.